client/ayon_hiero/plugins/publish/collect_plates.py

```diff
@@ -16,6 +17,17 @@ def process(self, instance):
         """
         instance.data["families"].append("clip")
 
+        # Adjust instance data from parent otio timeline.
+        otio_timeline = instance.context.data["otioTimeline"]
+        otio_clip, _ = utils.get_marker_from_clip_index(
+            otio_timeline, instance.data["clip_index"]
+        )
+        if not otio_clip:
+            raise RuntimeError(
```

Comment:
Requires import `from ayon_core.pipeline import PublishError`.
```suggestion
            raise PublishError(
```